packages/stdcompat/stdcompat.0/desc

```diff
@@ -0,0 +1 @@
+Compatibility module for OCaml standard library
```

Comment:
This is a very useful sounding library! A little more information from the README in the long description here would help making it discoverable.  There is also a typo: the `desc` file should be renamed to `descr` so it is picked up correctly.